lib/liquid/tags/include.rb

```diff
@@ -44,7 +43,10 @@ def parse(tokens)
 
     def render(context)
       partial = load_cached_partial(context)
-      variable = context.evaluate(@variable_name)
+
+      template_name = context.evaluate(@template_name_expr)
+      variable = context.evaluate(@variable_name_expr || Expression.parse(template_name))
```

Comment:
Hm. As I said on the other PR, do we really have to do this here? Calling `parse` in the `render` step feels kinda wrong.